Missionframework/modules/11_permission/fnc/fn_permission_openDialog.sqf

```diff
@@ -0,0 +1,116 @@
+#include "..\..\..\KPGUI\KPGUI_defines.hpp"
+/*
+    KPLIB_fnc_permission_openDialog
+
+    File: fn_permission_openDialog.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-12-14
+    Last Update: 2018-12-27
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Opens the permission dialog.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+if !(KPLIB_param_permission) then {
```

Comment:
```suggestion
 if !(KPLIB_param_permission) exitWith {
```

If not enabled the action is not showing (other suggestion), but just to catch the case that it's opened via debug console or something other strange way.